### PR TITLE
Player movement

### PR DESCRIPTION
**Pull request by Jeff-Moreau**
Player Moving Around is now done

### CODE REVIEW

File: Source/SteveSloth/Private/MyPlayer.cpp [hunk 1]
```diff
@@ -10,41 +10,64 @@
...
 }
 
+void AMyPlayer::MoveForwardBack(const FInputActionValue& Value)
+{
+	float const Direction = Value.Get<float>();
+	FVector const Forward = GetActorForwardVector();
+	AddMovementInput(Forward, Direction);
+}
+
+void AMyPlayer::TurnLeftRight(const FInputActionValue& Value)
```
Comment (Saretzky) [inline Source/SteveSloth/Private/MyPlayer.cpp:66]: Rename thins to MoveLeftRight and add a comment saying move and turn in that direction

File: Source/SteveSloth/Public/MyPlayer.h [hunk 2]
```diff
@@ -56,18 +41,33 @@ class STEVESLOTH_API AMyPlayer : public ACharacter
...
+	virtual void BeginPlay() override;
 	virtual void SetupPlayerInputComponent(class UInputComponent* PlayerInputComponent) override;
 
-};
+public:	
+	virtual void Tick(float DeltaTime) override; 
+
+private:
+	void MoveForwardBack(const FInputActionValue& Value);
+	void TurnLeftRight(const FInputActionValue& Value);
```
Comment (Saretzky) [inline Source/SteveSloth/Public/MyPlayer.h:72]: Rename thins to MoveLeftRight

File: Source/SteveSloth/Public/MyPlayer.h
Comment (Saretzky) [inline Source/SteveSloth/Public/MyPlayer.h:1]: Add P to all the pointers

**Review by Saretzky — CHANGES_REQUESTED** [review]

**Review by Saretzky — APPROVED** [review]
looks good

**Review by AMacLeod99 — APPROVED** [review]

File: Source/SteveSloth/Public/MyPlayer.h [hunk 3]
```diff
@@ -56,18 +41,33 @@ class STEVESLOTH_API AMyPlayer : public ACharacter
...
-	//Count Bosses Killed
-	int BossesKilled;
-	//Count Leaves Found
+	UPROPERTY(EditAnywhere, Category = "Input|Actions")
+	UInputAction* PJumping;
+	
+private:
 	int LeavesFound;
-	//Count Grubs Collected
     int GrubsCollected;
```
Comment (Guysus) [inline Source/SteveSloth/Public/MyPlayer.h:58]: not spaced the same

**Review by Guysus — CHANGES_REQUESTED** [review]

**Review by Guysus — APPROVED** [review]
good work

**Pull request merged**